tests/main/install-hook-can-run-svcs/test-snapd-install-hook-runs-svc/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-install-hook-runs-svc
+version: 1.0
+summary: A snap that starts a svc during the install hook
+description: A snap that starts a svc during the install hook
```

Comment:
Would adding "momentarily starts" better reflect the behavior of the hook?

Reply:
sure